### PR TITLE
Fixes nanopaste not using charge for surgery.

### PR DESCRIPTION
**Pull request by BoryaTheSlayer**
What it says in the name. Nanopaste will now actually lose charge if it's being used to repair internal robotic parts.

### CODE REVIEW

**Comment by BoryaTheSlayer** [comment]
!bugfix

**Comment by BoryaTheSlayer** [comment]
!review

File: code/modules/surgery/robotics.dm [hunk 1]
```diff
@@ -230,7 +230,7 @@
 					"You start mending the damage to [target]'s [I.name]'s mechanisms." )
 
 	target.custom_pain("The pain in your [affected.name] is living hell!", 75)
-	..()
+	..()
```
Comment (Doxxmedearly) [inline code/modules/surgery/robotics.dm]:
```suggestion
	..()
```
Your leftover spaces continue to haunt you

Comment (BoryaTheSlayer) [reply code/modules/surgery/robotics.dm]: rofl

File: code/modules/surgery/robotics.dm [hunk 2]
```diff
@@ -246,6 +246,10 @@
 				var/obj/item/organ/internal/brain/sponge = target.internal_organs_by_name[BP_BRAIN]
 				if(sponge && istype(I, sponge))
 					target.cure_all_traumas()
+				if(istype(tool, /obj/item/stack/nanopaste))
+					var/obj/item/stack/nanopaste/nanopaste = tool
+					nanopaste.use(1)
```
Comment (Arrow768) [inline code/modules/surgery/robotics.dm]: Is there a check somewhere to ensure that the nanopaste isnt empty already?

File: html/changelogs/boryatheslayer - nanopastefix.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+author: BoryaTheSlayer
+
+changes: 
+  - bugfix: "Fixes nanopaste not using charge when used for internal robotic repairs."
```
Comment (NonQueueingMatt) [inline html/changelogs/boryatheslayer - nanopastefix.yml]: This is missing `delete-after`.

Comment (BoryaTheSlayer) [reply html/changelogs/boryatheslayer - nanopastefix.yml]: Bruh. I'll fix.

File: code/modules/surgery/robotics.dm (hunk 2 above)
Comment (BoryaTheSlayer) [reply code/modules/surgery/robotics.dm]: No. I'll fix.

Comment (skull132) [reply code/modules/surgery/robotics.dm]: This isn't addressed. Do this in the `can_use` proc above.
```dm
	if (istype(tool, /obj/item/stack/nanopaste))
		var/obj/item/stack/nanopaste/NP = tool
		if (NP.uses_charge < 1)
			to_chat(user, span("notice", "[NP] is empty and cannot be used for this surgery."))
			return SURGERY_FAILURE
```

Comment (Nerrathiel) [reply code/modules/surgery/robotics.dm]: Not needed. Nanopaste cannot be empty. Even 'empty' sprite nanopaste still has 1 charge left. Upon being used up, the item deletes itself. There is no need to check for if it has charges left unless there's another purpose to it.

**Pull request merged**